Lib/fontTools/cffLib/cff2mergePen.py

```diff
@@ -0,0 +1,297 @@
+# Copyright (c) 2009 Type Supply LLC
+
+from __future__ import print_function, division, absolute_import
+from fontTools.misc.fixedTools import otRound
+from fontTools.misc.psCharStrings import CFF2Subr
+from fontTools.pens.t2CharStringPen import T2CharStringPen
+from fontTools.cffLib.specializer import (commandsToProgram,
+										  specializeCommands)
+from fontTools.cffLib import maxStackLimit
+
+
+class MergeTypeError(TypeError):
+	def __init__(self, point_type, pt_index, m_index, default_type, glyphName):
+		self.error_msg = [
+					"In glyph '{gname}' "
+					"'{point_type}' at point index {pt_index} in master "
+					"index {m_index} differs from the default font point "
+					"type '{default_type}'"
+					"".format(gname=glyphName,
+							  point_type=point_type, pt_index=pt_index,
+							  m_index=m_index, default_type=default_type)
+					][0]
+		super(MergeTypeError, self).__init__(self.error_msg)
+
+
+class CFF2CharStringMergePen(T2CharStringPen):
+	"""Pen to merge Type 2 CharStrings.
+	"""
+	def __init__(self, default_commands,
+				 glyphName, num_masters, master_idx, roundTolerance=0.5):
+		super(
+			CFF2CharStringMergePen,
+			self).__init__(width=None,
+						   glyphSet=None, CFF2=True,
+						   roundTolerance=roundTolerance)
+		self.pt_index = 0
+		self._commands = default_commands
+		self.m_index = master_idx
+		self.num_masters = num_masters
+		self.prev_move_idx = 0
+		self.glyphName = glyphName
+		self.roundTolerance = roundTolerance
+
+	def _round(self, number):
```

Comment:
instead of copying this code from the t2CharStringePen module, I'd suggest you move to a module-level scope (currently it's a closure nested in makeRoundFunc) import it from there

Reply:
Sounds like a good idea. Done.